### ISSUE TITLE
Web Search Alt Text

### ISSUE THREAD

**Issue by Metroid-Prime** [adirh3/Fluent-Search#35]
Web Search Alt Text should be split between the different columns to container

![image](https://user-images.githubusercontent.com/1836362/120909573-21545380-c62b-11eb-8a78-650638936822.png)

**Comment by Metroid-Prime** [comment]
Closing since #450 covers additional optimizations - this is not one of them, but the alt text has changed since the original post. 

![image](https://user-images.githubusercontent.com/1836362/155902601-b0ae48f4-f726-4687-83d8-5a33c3b0f911.png)
